EE_endpoints.md

```diff
@@ -0,0 +1,130 @@
+### Introduction
+Each request comes in through an endpoint. Endpoints handle the request up to the
+point of passing it off the a [Router](http://www.phoenixframework.org/docs/routing).
+
+Each request begins and ends it's lifecycle inside your application in an endpoint.
```

Comment:
These two intro lines are similar, maybe we could merge the two and have:

 > Each request begins and ends it's lifecycle inside your application in an endpoint. Endpoints handle the...